src/settings.py

```diff
@@ -166,7 +162,7 @@ def get_arg_specific_settings(args: Dict[str, Any]) -> Dict[str, Any]:
                 {
                     # In the playroom domain, HFF works better.
                     "playroom": "hff",
-                })[args["env"]],
+                })[args.get("env", "")],
```

Comment:
annoying that we can't get rid of this altogether, it's just here to make single unit tests work (both unit test suite and regular execution will always have env and approach set)